tests/driver_at86rf2xx/cmd.c

```diff
@@ -148,7 +148,8 @@ int txtsnd(int argc, char **argv)
 {
     char *text;
     uint8_t addr[_MAX_ADDR_LEN];
-    int iface, idx = 2, res;
+    int iface, idx = 2;
+    size_t res;
```

Comment:
Why did you need to change this? The parse function returns int

Reply:
Because of this:

```console
/Users/facosta/git/RIOT-OS/RIOT/tests/driver_at86rf2xx/cmd.c:160:36: error: comparison of integers of different signs: 'int' and
      'unsigned long' [-Werror,-Wsign-compare]
            if ((res <= 0) || (res > sizeof(pan))) {
                               ~~~ ^ ~~~~~~~~~~~
1 error generated.
```

Reply:
It would always make more sense to me to change the comparison than, instead of the type.

Reply:
Oh I just sah that `send()` below expects type `size_t` anyway (see remove of cast), so changing the type to `size_t` makes sense in the end, I think.

Reply:
Agree.